mscore/omrpanel.cpp

```diff
@@ -50,7 +50,7 @@ void MuseScore::showOmrPanel(bool visible)
 //---------------------------------------------------------
 
 OmrPanel::OmrPanel(QWidget* parent)
-   : QDockWidget(tr("OMR Panel"), parent)
+   : QDockWidget(tr("PDF Transcribing Assistant"), parent)
```

Comment:
Is this where the merge conflict Comes from, as that part had been merged earlier already?